
Due to MODULEPATH changes, the following have been reloaded:
  1) openmpi/2.1.1

Python 3.5.4
2019-05-24 23:07:30.170016: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1432] Found device 0 with properties: 
name: Tesla P100-PCIE-12GB major: 6 minor: 0 memoryClockRate(GHz): 1.3285
pciBusID: 0000:04:00.0
totalMemory: 11.91GiB freeMemory: 11.63GiB
2019-05-24 23:07:30.170176: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1511] Adding visible gpu devices: 0
2019-05-24 23:07:31.392307: I tensorflow/core/common_runtime/gpu/gpu_device.cc:982] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-05-24 23:07:31.392460: I tensorflow/core/common_runtime/gpu/gpu_device.cc:988]      0 
2019-05-24 23:07:31.392509: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1001] 0:   N 
2019-05-24 23:07:31.392911: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 11251 MB memory) -> physical GPU (device: 0, name: Tesla P100-PCIE-12GB, pci bus id: 0000:04:00.0, compute capability: 6.0)
2019-05-24 23:07:31.393510: I tensorflow/core/common_runtime/process_util.cc:69] Creating new thread pool with default inter op setting: 2. Tune using inter_op_parallelism_threads for best performance.
Using TensorFlow backend.
x path: ../PreprocessedData/preprocessedForCNN/299x299/imgData_(299, 299, 3)_train_n1000.npy
Done!
2019-05-24 23:08:33.466944: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1432] Found device 0 with properties: 
name: Tesla P100-PCIE-12GB major: 6 minor: 0 memoryClockRate(GHz): 1.3285
pciBusID: 0000:04:00.0
totalMemory: 11.91GiB freeMemory: 11.63GiB
2019-05-24 23:08:33.467073: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1511] Adding visible gpu devices: 0
2019-05-24 23:08:33.820569: I tensorflow/core/common_runtime/gpu/gpu_device.cc:982] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-05-24 23:08:33.820707: I tensorflow/core/common_runtime/gpu/gpu_device.cc:988]      0 
2019-05-24 23:08:33.820748: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1001] 0:   N 
2019-05-24 23:08:33.821085: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 11251 MB memory) -> physical GPU (device: 0, name: Tesla P100-PCIE-12GB, pci bus id: 0000:04:00.0, compute capability: 6.0)
2019-05-24 23:08:33.821590: I tensorflow/core/common_runtime/process_util.cc:69] Creating new thread pool with default inter op setting: 2. Tune using inter_op_parallelism_threads for best performance.
Using TensorFlow backend.
x path: ../PreprocessedData/preprocessedForCNN/299x299/imgData_(299, 299, 3)_val.npy
Done!
2019-05-24 23:09:09.446088: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1432] Found device 0 with properties: 
name: Tesla P100-PCIE-12GB major: 6 minor: 0 memoryClockRate(GHz): 1.3285
pciBusID: 0000:04:00.0
totalMemory: 11.91GiB freeMemory: 11.63GiB
2019-05-24 23:09:09.446254: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1511] Adding visible gpu devices: 0
2019-05-24 23:09:09.863073: I tensorflow/core/common_runtime/gpu/gpu_device.cc:982] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-05-24 23:09:09.863224: I tensorflow/core/common_runtime/gpu/gpu_device.cc:988]      0 
2019-05-24 23:09:09.863272: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1001] 0:   N 
2019-05-24 23:09:09.863672: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 11251 MB memory) -> physical GPU (device: 0, name: Tesla P100-PCIE-12GB, pci bus id: 0000:04:00.0, compute capability: 6.0)
2019-05-24 23:09:09.864278: I tensorflow/core/common_runtime/process_util.cc:69] Creating new thread pool with default inter op setting: 2. Tune using inter_op_parallelism_threads for best performance.
Using TensorFlow backend.
x path: ../PreprocessedData/preprocessedForCNN/299x299/imgData_(299, 299, 3)_test.npy
Done!
Exception ignored in: <bound method BaseSession.__del__ of <tensorflow.python.client.session.Session object at 0x2b8b24451cf8>>
Traceback (most recent call last):
  File "/home/amourav/tensorflow/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 738, in __del__
TypeError: 'NoneType' object is not callable
